_rules/element-lang-matches-default-language-off6ek.md

```diff
@@ -37,7 +37,7 @@ This rule applies to any [HTML element][] with a `lang` attribute for which all
 
 - the element is an [inclusive descendant][] in the [flat tree][] of a `body` element; and
 - the element is in a [document][] with a [content type][] of `text/html`; and
-- the element's `lang` [attribute value][] is a [valid language tag][]; and
+- the element's `lang` [attribute value][] has a [known primary language tag][]; and
```

Comment:
I like how it turns out. "**has** a primary tag" does emphasize nicely the partiality of the match 👍